test/node/api-versions.js

```diff
@@ -74,10 +73,7 @@ test.test('must handle specific versions', (test) => {
     });
 
     // v1
-    client = {
-      connectPolicy: new jstp.SimpleConnectPolicy(null, null, '1'),
-    };
-    jstp.net.connect(app.name, client, port, (error, conn) => {
+    jstp.net.connect(app.name + '@1', null, port, (error, conn) => {
```

Comment:
You know, that's the case for which passing app argument as an object, that I wanted to stay in this PR in https://github.com/metarhia/jstp/pull/231#discussion_r126700322, would come in handy.
This also goes for all of the similar occurrences in the tests.